backend/database/models/index.js

```diff
@@ -0,0 +1,39 @@
+const fs = require("fs");
```

Comment:
I would rename the file. From my point of view index file can be confused with backend/index file